resource/local-resource/src/main/kotlin/com/ritense/resource/service/LocalResourceService.kt

```diff
@@ -29,62 +29,66 @@ import java.util.UUID
 class LocalResourceService : ResourceService {
 
     override fun store(key: String, multipartFile: MultipartFile): Resource {
-        TODO("Not yet implemented")
+        TODO(NOT_YET_IMPLEMENTED_MSG)
```

Comment:
I think for messages like this, we should just mark it as a false positive in sonar. This is how the method was generated. If anything, we should just throw a proper exception as defined by Valtimo.